test/integration/test_task_run_diagnostic_reports.py

```diff
@@ -14,7 +14,10 @@ def mod_header(request):
 
 
 def test_run_diagnostic_reports():
-    versions = list(run_diagnostic_reports("testGroup", "testDisease").keys())
+    result = run_diagnostic_reports("testGroup",
+                                    "testDisease",
+                                    "estimate_uploader@example.com")
```

Comment:
Might be good to have 2 extra recipients for this test. At the moment this will work but adding an extra recipient would fail.